mtenv/core.py

```diff
@@ -147,7 +179,10 @@ def reset(self) -> ObsType:
             ObsType: For more information on `multitask observation`
             returned by the environment, refer :ref:`multitask_observation`.
         """
-        pass
+        if env_seed is not None:
+            self.np_random_env, env_seed = seeding.np_random(env_seed)
+        if task_seed is not None:
+            self.np_random_task, task_seed = seeding.np_random(task_seed)
```

Comment:
This looks off because this is just a partial implementation of `reset()`, that doesn't actually comply to the API since it doesn't return anything (which was also the case with the previous code btw).

It seems to me that `reset()` should raise a `NotImplementedError`, and these two lines moved to some utility method (e.g., `_maybe_seed_env_and_task()`) that would call `seed()` and `seed_task()` (rather than partially copying their implementation). This utility method could then be called from subclasses in their own implementation of `reset()`. This would be mentioned in the comment above, that needs to be updated since calling `seed()` before invoking `reset()` is not mandatory anymore.

Edit: although I just realized they do something similar in `gym`, and they are deprecating the `seed()` method... no time to give it more thought now I'm afraid